### PR TITLE
Add event name in Middlewares

### PR DESCRIPTION
**Pull request by Kahbazi**
#6381

### CODE REVIEW

File: src/Middleware/CORS/src/Internal/CORSLoggerExtensions.cs [hunk 1]
```diff
@@ -25,47 +25,47 @@ static CORSLoggerExtensions()
         {
             _isPreflightRequest = LoggerMessage.Define(
                 LogLevel.Debug,
-                1,
+                 new EventId(1, "IsPreflightRequest"),
```
Comment (khellang) [inline src/Middleware/CORS/src/Internal/CORSLoggerExtensions.cs]: Use `nameof(IsPreflightRequest)`? 🤔

Comment (khellang) [reply src/Middleware/CORS/src/Internal/CORSLoggerExtensions.cs]: Oh, nevermind. Just saw https://github.com/aspnet/AspNetCore/pull/6420#discussion_r245768445

Comment (Eilon) [inline src/Middleware/CORS/src/Internal/CORSLoggerExtensions.cs]: BTW the formatting seems a bit off - the new event id lines are indented one extra space?

Comment (Kahbazi) [reply src/Middleware/CORS/src/Internal/CORSLoggerExtensions.cs]: It's done.

File: src/Middleware/Localization/src/Internal/RequestCultureProviderLoggerExtensions.cs [hunk 2]
```diff
@@ -17,11 +17,11 @@ static RequestCultureProviderLoggerExtensions()
         {
             _unsupportedCulture = LoggerMessage.Define<string, IList<StringSegment>>(
                 LogLevel.Warning,
-                1,
+                 new EventId (1, "UnsupportedCulture"),
```
Comment (JamesNK) [inline src/Middleware/Localization/src/Internal/RequestCultureProviderLoggerExtensions.cs]: Some more indenting that is slightly off

File: src/Middleware/Localization/src/Internal/RequestCultureProviderLoggerExtensions.cs [hunk 3]
```diff
@@ -17,11 +17,11 @@ static RequestCultureProviderLoggerExtensions()
...
                 "{requestCultureProvider} returned the following unsupported cultures '{cultures}'.");
             _unsupportedUICulture = LoggerMessage.Define<string, IList<StringSegment>>(
                 LogLevel.Warning,
-                2,
+                 new EventId(2, "UnsupportedUICulture"),
```
Comment (JamesNK) [inline src/Middleware/Localization/src/Internal/RequestCultureProviderLoggerExtensions.cs]: ^

**Review by JamesNK — COMMENTED** [review]
A couple more small formatting issues then it is good to go.

**Comment by JamesNK** [comment]
Also a merge conflict to resolve.

**Review by JamesNK — APPROVED** [review]

**Pull request merged**